bazarr/api/movies/history.py

```diff
@@ -121,24 +119,36 @@ def get(self):
             'matches': x.matched,
             'dont_matches': x.not_matched,
             'external_subtitles': [y[1] for y in ast.literal_eval(x.external_subtitles) if y[1]],
-            'upgradable': bool(x.upgradable) if _language_still_desired(x.language, x.profileId) else False,
             'blacklisted': bool(x.blacklisted),
         } for x in database.execute(stmt).all()]
 
         for item in movie_history:
-            original_video_path = item['path']
-            original_subtitle_path = item['subtitles_path']
+            still_desired = True
+            if not _language_still_desired(item['language'], item['profileId']):
+                # this language isn't desired anymore so there's no point in upgrading this subtitles
+                still_desired = False
+
             item.update(postprocess(item))
 
+            # Mark upgradeable and get original_id
+            item.update({'original_id': upgradable_movies_not_perfect.get(item['id'])})
+            language = item['language']['code2']
```

Comment:
I'm sure to be missing something, but where is this variable being used? Is there any kind of reflection magic?

Reply:
It's not used anymore since I've added still_desired boolean. Good catch!